book_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../book'
+require_relative '../rental'
+require_relative '../person'
```

Comment:
- [ ] @Team Great work implementing the tests for the book class, however, you have imported the book rental and person class wrongly, it should be with a single dot since they are in the same directory eg instead of `require_relative '../book'` it should be `require_relative './book'`